File: generated-code/python/codecraft/FileReadWrite/model/repair_action.py
class RepairAction:
    """Repair action"""

    def __init__(self, target):
        self.target = target
        """Target entity's ID"""

    @staticmethod
    def read_from(stream):
        """Read RepairAction from input stream
        """
        target = stream.read_int()
        return RepairAction(target)

    def write_to(self, stream):
        """Write RepairAction to output stream
        """
        stream.write_int(self.target)

    def __repr__(self):
        return "RepairAction(" + \
            repr(self.target) + \
            ")"

File: generated-code/javascript/codecraft/FileReadWrite/model/repair-action.js
/**
 * Repair action
 */
class RepairAction {
    /**
     * Target entity's ID
     */
    target;

    constructor(target) {
        this.target = target;
    }

    /**
     * Read RepairAction from input stream
     */
    static async readFrom(stream) {
        let target;
        target = await stream.readInt();
        return new RepairAction(target);
    }

    /**
     * Write RepairAction to output stream
     */
    async writeTo(stream) {
        let target = this.target;
        await stream.writeInt(target);
    }
}
module.exports = RepairAction